src/sentry/options/defaults.py

```diff
@@ -1817,6 +1817,12 @@
 # Decides whether an incoming span triggers an update of the clustering rule applied to it.
 register("span_descs.bump-lifetime-sample-rate", default=0.25, flags=FLAG_AUTOMATOR_MODIFIABLE)
 
+# === Nodestore related runtime options ===
+
+register(
+    "nodestore.set_subkeys.enable_set_cache_item", default=False, flags=FLAG_AUTOMATOR_MODIFIABLE
```

Comment:
Do it the other way around so that the rollout retains the current behavior. Then turn on with the options to see if it has the desired effect before making it permanent/flipping the default.

Also the recommended pattern is dashes instead of underscores for options.